File: app/src/main/java/com/example/myapplication/model/Laudo.java
package com.example.myapplication.model;

public class Laudo {

    private int id;
    private String nome;
    private String descrição;
    private int gestante;
    private int idData;

    public Laudo(String nome, String descrição, int gestante, int idData) {
        this.nome = nome;
        this.descrição = descrição;
        this.gestante = gestante;
        this.idData = idData;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNome() {
        return nome;
    }

    public void setNome(String nome) {
        this.nome = nome;
    }

    public String getDescrição() {
        return descrição;
    }

    public void setDescrição(String descrição) {
        this.descrição = descrição;
    }

    public int getGestante() {
        return gestante;
    }

    public void setGestante(int gestante) {
        this.gestante = gestante;
    }

    public int getIdData() {
        return idData;
    }

    public void setIdData(int idData) {
        this.idData = idData;
    }
}

File: SERVIDOR/Model/Laudo.php
<?php

/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

/**
 * Description of Laudo
 *
 * @author luan
 */
class Laudo {
    private $id;
    private $nome;
    private $descricao;
    private $idRegistro;
    private $gestante;
    private $idData;
    
    function __construct($nome, $descricao, $idRegistro, $gestante, $idData) {
        $this->nome = $nome;
        $this->descricao = $descricao;
        $this->idRegistro = $idRegistro;
        $this->gestante = $gestante;
        $this->idData = $idData;
    }
    
    function getId() {
        return $this->id;
    }

    function getNome() {
        return $this->nome;
    }

    function getDescricao() {
        return $this->descricao;
    }

    function getIdRegistro() {
        return $this->idRegistro;
    }

    function getGestante() {
        return $this->gestante;
    }

    function getIdData() {
        return $this->idData;
    }

    function setId($id) {
        $this->id = $id;
    }

    function setNome($nome) {
        $this->nome = $nome;
    }

    function setDescricao($descricao) {
        $this->descricao = $descricao;
    }

    function setIdRegistro($idRegistro) {
        $this->idRegistro = $idRegistro;
    }

    function setGestante($gestante) {
        $this->gestante = $gestante;
    }

    function setIdData($idData) {
        $this->idData = $idData;
    }

}
